src/plugins/internalnotification/README.md

```diff
@@ -0,0 +1,37 @@
+- infos = Plugin for internal notification
+- infos/author = Thomas Wahringer <waht@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage postcommit
+- infos/status = global libc unittest concept unfinished experimental nodoc
+- infos/metadata =
+- infos/description = Plugin for internal notification
+
+## Usage ##
+
+Allows programs to register integer variables that get automatically updated
+when the value of the key changes.
+
+Use as global plugin:
+$ sudo kdb global-mount internalnotification
+
+## Exported Methods ##
+
+This plugin exports the function `int elektraInternalnotificationRegisterInt (Plugin * handle, int * variable, Key * key)`.
```

Comment:
How would a user get a handle to the plugin? You should write a wrapper that takes KDB and does the lookup to your plugin's handle.